packages/ndla-ui/src/locale/messages-en.js

```diff
@@ -553,6 +553,11 @@ const messages = {
     save: 'Save',
     abort: 'Abort',
   },
+  codeBlock: {
+    copiedCode: 'Copied code',
+    copyButton: 'Copy code',
+    copyCode: 'Copy code',
+  },
```

Comment:
Du la de translations til i ndla-ui men ser ikke noe ndla-ui dependencies i ndla-code package.json, kommer det til å fungere?

Reply:
La i dag merke til at alle steder denne pakken vil bli brukt så henter messages fra @ndla/ui i tillegg til egne oversettelsesfiler.

Reply:
Det betyr jo at du må ha ndla-ui som avhengighet også for at denne skal fungere?

Reply:
ja, for at tekststrengene skal fungere. 

De lastes inn sånn som dette, https://github.com/NDLANO/editorial-frontend/blob/master/src/i18n.js#L9

Et alternativ var å opprette nye `locale/messagesNB, locale/messagesNN, locale/messagesEN`filer i ndla-code. Da må alle stedene som bruker `@ndla/code` få oppdatert sinn `i18n.js`/` messages.js` med hvor de henter teksten fra (på samme måte som i linken over her)

Reply:
Det ryddigeste hadde nok å ha språkfiler i pakka, men den vil jo aldri bli brukt utenfor kontekst av ndla-ui så det er ikkje så farlig.